examples/feature_demo/fly_controller_logo.py

```diff
@@ -0,0 +1,196 @@
+"""
```

Comment:
maybe this example should just be renamed to something else. I don't like how I've highjacked the `fly_controller.py` name making that one harder to press tab onto.

I wanted to show:

1. Customizing of points with custom SDF.
2. Overlaying of your own custom logo onto your scene.

Reply:
i've renamed the logo